flag/controlplane/controlplane.go

```diff
@@ -0,0 +1,9 @@
+package controlplane
+
+// ControlPlane represents control plane specific configuration,
+// used for templating resources in tenant clusters.
+type ControlPlane struct {
+	// WorkerSubnets is comma-separated list of control-plane
+	// workers subnets.
+	WorkerSubnets string
```

Comment:
Might be a matter of taste, but to align with our usual style, I would maybe break this into `Worker` subtype that has field `Subnets`.

Reply:
I might cut it to just subnets, as on-prem we don't have that split on master/worker subnets